integration-tests/tests/webvitals/webvitals.spec.js

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module.exports = {
+  'webvitals spans': async function(browser) {
+    // the google webvitals library only works on chrome (and chrome-based edge) at the moment
+    const SUPPORTED_BROWSERS = ['Chrome', 'chrome', 'Edge'];
+    const currentBrowser = browser.options.desiredCapabilities.browserName;
+    if (!SUPPORTED_BROWSERS.includes(currentBrowser)) {
+      return;
+    }
+
+    const url = browser.globals.getUrl('/webvitals/webvitals.ejs');
+    await browser.url(url);
+    // fid won't happen unless there is interaction, so simulate a bit of that
+    await browser.click('#clicky');
+    await browser.waitForElementPresent('#p2');
+
+    await browser.execute(function() {
```

Comment:
Can you please add a comment that this is a workaround for headless Chrome not emitting `visibilitychange`?

Reply:
Good catch, will do.